### ISSUE TITLE
Backup DB

### ISSUE THREAD

**Issue by MosheLavee** [ajwecker/TS1218#79]
@TsvikaKuflik will check with Yaron. Should be performed at least few times in a day.

**Comment by dstoekl** [comment]
So far I have done backups with a simple manual sql dump. Have you installed any automatic system?

**Comment by TsvikaKuflik** [comment]
Yes,There is backup mechnism.Coordinated by UriTsvika on mobileבתאריך 21 בפבר׳ 2019 9:39,‏ dstoekl <notifications@github.com> כתב:So far I have done backups with a simple manual sql dump. Have you installed any automatic system?
